Source/DfBAdminToolkit.Common/Services/MemberServices.cs

```diff
@@ -145,6 +145,10 @@ public IServiceResponse AddMember(IMemberData data, string authToken)
                         ),
                         new JProperty("force_async", false)
                     );
+                    if (data.PersistentId != null) {
+                        JObject prov = (JObject)(jsonProv["new_members"] as JArray).First;
+                        prov.Property("member_surname").AddAfterSelf(new JProperty("member_persistent_id", data.PersistentId.Trim()));
```

Comment:
Does order actually matter here? It'd be cleaner if we can just do prov.Add

Reply:
done. Changed to prov.add